docs/index.adoc

```diff
@@ -592,6 +592,115 @@ public class SecuredServiceEndpoint {
 
 For more details see this guide: https://openliberty.io/guides/microprofile-jwt.html[Securing microservices with JWTs]
 
+== Messaging
+
+Applications can pass messages among themselves with the Java Message Service (JMS) APIs. Liberty can act as a server of JMS destinations (queues and topics,) or it can connect to another server to access those destinations--either another Liberty server or IBM MQ.
+
+=== JMS Server
+Liberty can act as a JMS server whether or not it is serving any applications. 
+
+.server.xml
+[source,xml]
+----
+<featureManager>
+    <feature>wasJmsServer-1.0</feature>
+</featureManager>
+
+<messagingEngine>
+    <queue id="libertyQueue" />
```

Comment:
for the IDs in the config snippets, lets use IDs such as `myQueue` and `myTopic`, which I think helps convey that the values don't have any special meaning and that the reader can use whatever value they see fitting